docs/contents/releases.md

```diff
@@ -1,128 +1,159 @@
-building into ACK.
+## Stable Helm Charts
 
-There will be a Github Issue for tracking the build of the ACK service
-controller for the service.
+We [recommend][recommend-helm] using Helm to install an ACK service controller.
 
-The GitHub Issue **WILL BE** associated with a GitHub Milestone indicating the
-target date for the `DEVELOPER PREVIEW` release of that service's controller.
+[recommend-helm]: https://aws-controllers-k8s.github.io/community/user-docs/install/#helm-recommended
 
-## BUILD
+If you want to only install ACK service controllers that have been determined
+to be stable by the ACK contributor team and the associated AWS service team,
+you may choose to install the Helm chart for an ACK service controller that has
+been tagged with the `-stable` suffix -- e.g. `elasticache-stable`.
 
-The `BUILD` status indicates that the ACK service controller for the AWS
-service is **actively being built** in preparation for `DEVELOPER PREVIEW`
-release of that ACK service controller.
+This `-stable` artifact tag points to a Helm chart that has configuration values that
+have been tested with a specific SemVer Docker image for the ACK service
+controller. Typically these tests are "soak" tests and allow the team
+maintaining that ACK controller's source code to have a high degree of
+confidence in the controller's long-running operation.
 
-It is in the `BUILD` status that we will identify those **AWS service API
-resources** that will be supported in `DEVELOPER PREVIEW` and which will be
-supported by the `GENERALLY AVAILABLE` release of the controller.
+!!! "important"
+    Please note that not all ACK service controllers have Helm chart that have
+    been tagged with the `-stable` suffix.
 
-!!! note "what do we mean by 'AWS service API resources'?
-    An *AWS service API resource* is a top-level object that can be created by
-    a particular AWS service API. For example, an SNS Topic or an S3 Bucket.
-    Some service APIs have multiple top-level resources; SNS, for instance, has
-    Topic, PlatformApplication and PlatformEndpoint top-level resources that
-    may be created.
+## Maintenance Phases
 
-## DEVELOPER PREVIEW
+As noted above, individual ACK service controllers all use Semantic Versioning
+("X.Y.Z") in order to signal breaking interface changes. However, each
+controller follows its own release cadence and each controller has a separate
+team of contributors that maintain the code, test the controller and determine
+whether the controller is stable in long-running operation.
 
-The `DEVELOPER PREVIEW` status indicates that the **source code and some
-documentation** for the ACK service controller for the AWS service has been
-check into the ACK source repository along with **minimual end-to-end test
-cases** that run against a local Kubernetes-in-Docker (KinD) cluster.
+ACK service controllers having release tags within a major Semantic Version
+("X") will be in one of four Maintenance Phases:
```

Comment:
Does it mean that these phases apply if major Semantic Version ("X") is non-zero?

Reply:
No, even zero major series can be GA. GA refers to our commitment to support the released code. the zero version refers to whether new releases in that major series can break backwards compatibility. Two different concepts and signals to the consumer.